src/CSharpMongoMigrations.Tests/MigrationRunnerTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using MongoDB.Driver;
+using Xunit;
+
+namespace CSharpMongoMigrations.Tests
+{
+    public class MigrationRunnerTests
+    {
+        [Theory]
+        [InlineData("mongodb://test:27017/db", "System")]
+        public void ShouldCreateMigrationRunnerWithMongoUrl(string connectionString, string assembly)
+        {
+            var url = MongoUrl.Create(connectionString);
+            var runner = new MigrationRunner(url, assembly);
```

Comment:
This test is useless for CI purpose. It might be helpful during development process but you don't need to include it to the final change set. Could you please remove it.